app/grandchallenge/verifications/models.py

```diff
@@ -131,6 +139,30 @@ def send_verification_email(self):
             user_email_override={self.user: self.email},
         )
 
+    def handle_pending_requests(self):
```

Comment:
This method name is misleading and should be changed. What is it doing? It is accepting pending requests for verified users. It should probably double check that the user is verified (or move that check to within the function call) before doing anything else in case a future developer calls it on the wrong object.